software/java/en.md

```diff
@@ -14,42 +14,42 @@ You can set up Java by following the instructions below:
 
 ## JRE
 
-Grab the Java Runtime Environment (JRE) as `.tar.gz` from the [Oracle Download Page](https://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html).
+Grab the newest Java Runtime Environment (JRE) as `.tar.gz` from the [Oracle Download Page](https://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html).
 
 Extract JRE and move it to /opt:
 
 ``` bash
 cd ~/Downloads
-tar xf jre-8u181-linux-x64.tar.gz
+tar xf jre-8u202-linux-x64.tar.gz
 sudo mkdir -p /opt
-sudo mv jre1.8.0_181 /opt/
-sudo ln -svf /opt/jre1.8.0_181/bin/java /usr/bin/java
+sudo mv jre1.8.0_202 /opt/
+sudo ln -svf /opt/jre1.8.0_202/bin/java /usr/bin/java
 ```
 
 To enable Java in Firefox:
 
 ``` bash
 mkdir -p ~/.mozilla/plugins
-ln -s /opt/jre1.8.0_181/lib/amd64/libnpjp2.so ~/.mozilla/plugins/libnpjp2.so
+ln -s /opt/jre1.8.0_202/lib/amd64/libnpjp2.so ~/.mozilla/plugins/libnpjp2.so
```

Comment:
Should just try using wildcards / globbing instead.

Reply:
On second thought, and after trying it again, I've removed the Firefox instructions completely, since they don't work anymore anyway (no support from Firefox for the Java plugin since September 2018)